apps/solana_app/solana_helpers.h

```diff
@@ -0,0 +1,61 @@
+/**
+ * @file    solana_helpers.h
+ * @author  Cypherock X1 Team
+ * @brief   Utilities api definitions for Solana chains
+ * @copyright Copyright (c) 2023 HODL TECH PTE LTD
+ * <br/> You may obtain a copy of license at <a href="https://mitcc.org/"
+ * target=_blank>https://mitcc.org/</a>
+ */
+#ifndef SOLANA_HELPERS_H
+#define SOLANA_HELPERS_H
+
+/*****************************************************************************
+ * INCLUDES
+ *****************************************************************************/
+
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+#include "coin_utils.h"
+
+/*****************************************************************************
+ * MACROS AND DEFINES
+ *****************************************************************************/
+
+#define SOLANA_PUB_KEY_SIZE 33
+#define SOLANA_PURPOSE_INDEX 0x8000002C
+#define SOLANA_COIN_INDEX 0x800001F5
+#define SOLANA_ACCOUNT_INDEX 0x80000000
+#define SOLANA_CHANGE_INDEX 0x80000000
```

Comment:
Also include the reference which was defined in coin_support/solana.c

Reply:
the references were added. resolving.